### DISCUSSION TITLE
Ability to export and import settings

### DISCUSSION THREAD

**Discussion by BlackRule** [valinet/ExplorerPatcher#639] [Ideas]
I know that some settings are in HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher but there are also other settings all around the registry. Would be so great to export everything available through EP settings GUI and then on several computers/user_profiles press "import settings" button and EP would act as if I manually set the settings. I could try to implemment it on my own and create pull_request but I doubt I'll handle c++) Should I try or you'll implemment it in the future because it's in your roadmap?

**Comment by valinet** [comment]
If you want to work on it, that's fine by me. I am a bit busy with work lately, don't have that much time for this. Contributions are always welcome. Make a fork, and post some updates here, maybe. Shouldn't be too hard or take too long to develop, would be a fun weekend project. Thank you.
